stream-chat-android/src/main/java/com/getstream/sdk/chat/utils/DateFormatter.kt

```diff
@@ -7,6 +7,7 @@ import java.util.Date
 
 public interface DateFormatter {
     public fun formatDate(localDateTime: LocalDateTime?): String
+    public fun formatTime(localDateTime: LocalDateTime?): String
```

Comment:
This should take a `LocalTime?` as its parameter

Reply:
Judging by the implementation, both these methods format the same thing but with different output